### ISSUE TITLE
stated performance of DGCNet cannot meet

### ISSUE THREAD

**Issue by Anguliachao** [lxtGH/GALD-Net#15]
Dear author,
   Thank you for publish this repo. I tried with original training/validation settings , as stated in readme, the resnet50 DualGCN for cityscape shall reach ~79%, However mine only reaches 67%
The details are attached as below: 

{"meanIU": 0.6755697115491832, "IU_array": [0.9699699360909015, 0.7804632423588875, 0.909098167476979, 0.47001360372795786, 0.53806631978281, 0.5845193952059451, 0.5983268985770329, 0.7145573340347664, 0.9099576676527088, 0.5301010366577027, 0.9328700787203849, 0.7606536183056566, 0.524505490879149, 0.9179202810460247, 0.45918543000315803, 0.6450625060757722, 0.39362567755342187, 0.49597250072955573, 0.7009553345556664]}

While I havenot change any settings in the original repo, my pytorch version is 1.6.0.
Could you explain a little bit why this performance gap is so large?

**Comment by lxtGH** [comment]
My pytorch version is 1.3.1+cuda90_cudnn7.6.3_lms. You should instanll apex from the source. 
The ckpt is reproduced by this repo with no modification shown in the readme.
What is the final loss of you model?

**Comment by Anguliachao** [comment]
hi @lxtGH ,
 That's right, the compilation of apex is compiled from the source as guided on official repo, which is no problem(cause pytorch cuda version shall match my nvcc -V in use cuda version, which are both 10.1) , and the loss as below:

2020-09-10 04:57:05,378 INFO    [train_distribute.py, 248] iter = 50571 of 50575 completed, lr=2.0338161418814677e-06, loss = 0.2640952169895172
2020-09-10 04:57:08,174 INFO    [train_distribute.py, 248] iter = 50572 of 50575 completed, lr=1.5698813385226603e-06, loss = 0.26161420345306396
2020-09-10 04:57:10,973 INFO    [train_distribute.py, 248] iter = 50573 of 50575 completed, lr=1.0898950842718759e-06, loss = 0.3417670726776123
2020-09-10 04:57:13,771 INFO    [train_distribute.py, 248] iter = 50574 of 50575 completed, lr=5.840603141371525e-07, loss = 0.14752346277236938
2020-09-10 04:57:15,539 INFO    [train_distribute.py, 256] Training cost: 141548.4647937687seconds
2020-09-10 04:57:15,539 INFO    [train_distribute.py, 257] Save final models

**Comment by lxtGH** [comment]
It is strange. I didn't meet this problem.

**Comment by Anguliachao** [comment]
hi, finally I retrain and made it to 76%, which should be fine. Thank you

**Comment by lxtGH** [comment]
@Anguliachao Hi a possible problems is  that you may not load image pretrained model ?
